ci/configs/gnu32.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+#
+# Copyright (c) The Bitcoin Core developers
```

Comment:
In commit "ci: add copyright to bash scripts" (fefee4bcdd739b62f9c98df4706b3b5c5ab47087)

Again would prefer to drop copyright from the config fragments, but keep it in the CI scripts. IMO copyright blends pretty well into source code but I find it jarring when it's injected into things like minor pieces of documentation and config files, and would prefer keep distractions to a minimum in places like that. Ideally these files can just begin with `CI_DESC="Description of CI job"`